src/test/java/cow/farm/utils/QuantityCheckingListTest.java

```diff
@@ -19,10 +16,10 @@ public void shouldReturnMapOfValues() {
 		list.add("b");
 		list.add("d");
 		//then
-		HashMap map = list.getQuantityOfEachObject();
-		Assert.assertEquals(map.get("a"), 2);
-		Assert.assertEquals(map.get("b"), 2);
-		Assert.assertEquals(map.get("c"), 1);
-		Assert.assertEquals(map.get("d"), 1);
+		//HashMap map = list.getQuantityOfEachObject();
```

Comment:
Poprawić to czy skasować bo to plik do poprzedniego zadania?
Czasem się zastanawiam, czy takie testy powinny się plątać w kodzie jak już nie są potrzebne a klasa do której się odnosiły się zmieniła.